src/FantasyFun.API/ViewModel/TeamName.cs

```diff
@@ -4,11 +4,18 @@ public class TeamName
     {
         public string LongName { get; }
         public string ShortName { get; }
+        public long Overall { get; }
```

Comment:
Do we need that overall for the team?

Reply:
We don't need it. That property and the second constructor in the "TeamName" class appeared just for test. I will delete them.